modules/references/client/components/Info.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// TODO we can't use the config/config for some reason
+import * as config from '@/config/env/default';
```

Comment:
We could perhaps make API service which simply exposes `window.apiSettings`, much like `settings` Angular service does:

https://github.com/Trustroots/trustroots/blob/f6893632a696d56ecfe64696313ac025112e0169/modules/core/client/services/settings.client.service.js#L48-L49

In future, this might then load settings via API and store in app state somehow.

Reply:
Do you mean we could have `GET /api/settings?fields=timeToReplyReference,...` will give the needed config value(s) from API?

Reply:
I think I'll hardcode the value for now with a TODO.
Creating the API endpoint later should be easy. I'm attracted by that idea. It makes the app more decoupled from API.

Reply:
Good call to hardcode it for now.

API would probably just return all the settings at once and store it in the app state to avoid multiple round-trips.